to-test.md

```diff
@@ -1,67 +1,52 @@
-## 4.1
+## 4.3: a new interface with easier-to-manage settings and features.
 
-### SSO
+With Jetpack 4.3, we completely revamped the admin interface. The new dashboard is faster, fully responsive, and offers you all the tools you need to monitor your site's health and performance.
 
-- **We've completely redesigned and refactored Jetpack's [SSO Module](https://jetpack.com/support/sso/).** It would be great if you could use the module as much as possible, with different users, in different browsers, and different roles on your site. The additional options mentioned at the bottom of [this page](https://jetpack.com/support/sso/) should still work.
+We'd like to you to try and test every aspect of that new interface.
 
-### Comments
+**To get started, use the "Disconnect Jetpack" link at the bottom of the Jetpack menu, and then use the "Reset Options" to start from scratch, as a brand new Jetpack user.**
```

Comment:
I'd like to drop this line-- people upgrading their weird configs are probably more valuable than people testing with fresh sites, don't you think @jeherve ?